### ISSUE TITLE
The SVG picture of the navigation menu ICON does not display correctly on Windows

### ISSUE THREAD

**Issue by HuaFangYun** [dotnet/maui#21622]
### Description

Ａｎｄｒｏｉｄ 
![image](https://github.com/dotnet/maui/assets/30250585/ca3904b2-1ea0-4668-a2c6-874b9f95a4ac)
Ｗｉｎｄｏｗｓ　
![image](https://github.com/dotnet/maui/assets/30250585/1716a7b6-39bf-44dd-bbf7-e3cebc688abe)
Demo
[MauiApp1.zip](https://github.com/dotnet/maui/files/14853654/MauiApp1.zip)

### Version with bug

8.0.10 SR3

### Is this a regression from previous behavior?

Yes, this used to work in .NET MAUI

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

net8.0-windows10.0.19041.0

**Comment by XamlTest** [comment]
Verified on VS 17.10.0 Preview 3.0(8.0.20). Repro on Windows 11, not repro on Android 14.0-API34 with below Project:

[MauiApp1.zip](https://github.com/dotnet/maui/files/14955080/MauiApp1.zip)

**Comment by HuaFangYun** [comment]
Right. The Svg menu picture cannot be displayed properly on Win11.

**Comment by HuaFangYun** [comment]
<Image
     Source="mysql.svg"
     HeightRequest="185"
     Aspect="AspectFit"
     SemanticProperties.Description="dot net bot in a race car number eight" />
     
     The image of svg does not display properly.

 <Button
     x:Name="CounterBtn" ImageSource="mysql.svg"
     Text="Click me" 
     SemanticProperties.Hint="Counts the number of times you click"
     Clicked="OnCounterClicked"
     HorizontalOptions="Fill" />

On win11. There seems to be something wrong with the svg display.

**Comment by sheiksyedm** [comment]
@HuaFangYun You can change the **.svg** with the **.png** extension in the code to resolve this issue. 

`<TabBar>
        <ShellContent Title="Home"
                Icon="home.png"
                ContentTemplate="{DataTemplate local:MainPage}"
                Route="MainPage"/>
        <ShellContent Title="NewPage1"
                Icon="mysql.png"
                ContentTemplate="{DataTemplate local:NewPage1}"
                Route="NewPage1"/>
    </TabBar>`

Closing as duplicate
https://github.com/dotnet/maui/issues/7066
